planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -337,7 +341,15 @@ void AvoidanceModule::fillAvoidanceTargetObjects(
     objects.push_back(createObjectData(data, object));
   }
 
-  utils::avoidance::filterTargetObjects(objects, data, debug, planner_data_, parameters_);
+  filterTargetObjects(objects, data, debug, planner_data_, parameters_);
+
+  helper::avoidance::AvoidanceHelper helper(planner_data_, parameters_);
+
+  const auto feasible_stop_distance = helper.getFeasibleDecelDistance(0.0);
```

Comment:
// Calculate the distance needed to safely decelerate the ego vehicle to a stop line.

Reply:
Add comment line in https://github.com/autowarefoundation/autoware.universe/pull/3978/commits/0d099f4f4baaa6ff9134f77160612075a7b5c162.